src/components/Map/Map.tsx

```diff
@@ -98,6 +103,10 @@ function Map() {
           <div>
             <h2>{activeMarket.market_name}</h2>
             <p>{activeMarket.address}</p>
+            <p>{activeMarket.phone}</p>
+            <a href={activeMarket.website} target="_blank" rel="noopener noreferrer">
```

Comment:
I love this feature!  Letting the user be able to click on a link and the Farmer's Market website opens, good stuff.  We're talking right now about if we should add labels to the popup??  Maybe, although these might be obvious.  We'll spin it around and decide.